content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md

```diff
@@ -0,0 +1,149 @@
+    ```
+    Modify the `net-conf.json` section of the flannel manifest in order to set the VNI to 4096 and the Port to 4789. It should look as follows:
+
+    ```powershell
+    net-conf.json: |
+    {
+      "Network": "10.244.0.0/16",
+      "Backend": {
+        "Type": "vxlan",
+        "VNI" : 4096,
+        "Port": 4789
+      }
+    }
+    ```
+
+{{< note >}}
+The VNI must be set to 4096 and port 4789 for Flannel on Linux to interoperate with Flannel on Windows. See the [VXLAN documentation](). for an explanation of these fields.
+{{< /note >}}
+
+{{< note >}}
+To use L2Bridge/Host-gateway mode instead change the value of `Type` to `"host-gw"` and omit `VNI` and `Port`.
+{{< /note >}}
+
+3.  Apply the Flannel manifest and validate
+    
+    Let's apply the Flannel configuration:
+
+    ```powershell
+    kubectl apply -f kube-flannel.yml
+    ```
+    After a few minutes, you should see all the pods as running if the Flannel pod network was deployed.
+
+    ```powershell
+    kubectl get pods -n kube-system
+    ```
+
+    The output should include the Linux flannel DaemonSet as running:
+
+    ```shell
+    NAMESPACE     NAME                                      READY        STATUS    RESTARTS   AGE
+    ...
+    kube-system   kube-flannel-ds-54954                     1/1          Running   0          1m
+    ```
+4.  Add Windows Flannel and kube-proxy DaemonSets
+
+    Now you can add Windows-compatible versions of Flannel and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v{{< skew currentVersion >}}, but you should adjust the version for your own deployment.
+
+    ```powershell
+    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v{{< skew currentVersion >}}/g' | kubectl apply -f -kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+    ```
+
+{{< note >}}
+If you're using host-gateway use https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-host-gw.yml instead
+{{< /note >}}
+
+  {{< note >}}
+If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line: `powershell wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"` in the `flannel-host-gw.yml` or `flannel-overlay.yml` file and specify your interface accordingly. `bash # Example curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml | sed 's/Ethernet/Ethernet0 2/g' | kubectl apply -f -`
+{{< /note >}}
+
+### Joining a Windows worker node
+
+
+{{< note >}}
+You must install the `Containers` feature and install Docker. Instructions to do so are available at [Install Docker Engine - Enterprise on Windows Servers]().
```

Comment:
Forget to add url in ()?

Reply:
Thank you for your feedback! I appreciate your attention to detail. I will now add the missing URL in parentheses. Please let me know if there's anything else you'd like me to address.